forestserviceprototype/specialuseform/templates/specialuseform/home.html

```diff
@@ -0,0 +1,14 @@
+{% extends "specialuseform/base.html" %}
+
+{% block content %}
+<h1>Welcome to the Forest Service Online Permits System</h1>
+
+<p>Are you an commerical outfitter?</p>
```

Comment:
Maybe better to put this after the non-commercial group use permit and phrase it as **"Need to apply for another kind of permit?"** (Many groups, including, but not exclusively, commercial outfitters, need to apply for special use permits.)